src/Scenes/GameScene.cs

```diff
@@ -76,7 +82,7 @@ public override void OnStart(Dictionary<string, object> args)
         
         game.LineClear += (e) =>
         {
-            Audio.PlayOneShotAsync(Resources.GetLineClearSound(e));
+            Audio.PlayOneShotAsync(Resources.SfxLineClear);
```

Comment:
実装が古いのに戻ってしまっているので戻してください

```suggestion
            Audio.PlayOneShotAsync(Resources.GetLineClearSound(e));
```

Reply:
897eb0のバージョンを基にして書いてしまっていたので、修正するとResourceLoaderのエラーが出てしまいます。
一旦そのままにしています。